events/forms.py

```diff
@@ -1268,6 +1269,60 @@ class Meta:
     category = ModelChoiceField(queryset=Category.objects.all(), required=False)
     service = ModelChoiceField(queryset=Service.objects.all(), required=False)  # queryset gets changed in constructor
 
+class MKSingleHoursForm(forms.ModelForm):
```

Comment:
If I remember correctly, there was already a view for this so creating this form may be redundant.

Reply:
The only [single-instance hour form](https://github.com/WPI-LNL/lnldb/blob/0dc50dc72f7d2d8a48daf414a9f74893887d39bc/events/views/my.py#L205) that I could find is tied to an individual user and based on Event2012. I based this one off of the bulk hours one and I could combine them, but I figured this would be easier.